pages/common/mediainfo.md

```diff
@@ -0,0 +1,15 @@
+# mediainfo
+
+> Display metadata from video and audio files.
+
+- Display metadata for a given file in the console:
+
+`mediainfo {{file}}`
+
+- Store the output to a given file apart from displaying in the console:
+
+`mediainfo --Logfile={{out.txt}} {{file}}`
+
+- Display the list of metadata attributes that can be extracted:
```

Comment:
This is useful - it seems to give a list of parameters and then a description for each. Different file types may have different metadata attributes that you can extract I assume though. Is there a way to list them for a specific file type too?

Reply:
Ah no. I didn't find anything like that.

Reply:
Shame! Just a thought, anyway.